cluster-scope/overlays/nerc-ocp-infra/machineconfigs/configure-bond0/src/configure-bond0-ports.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+mkdir -p /etc/NetworkManager/system-connections-disabled
+mv /etc/NetworkManager/system-connections/ens*.nmconnection /etc/NetworkManager/system-connections-disabled/
```

Comment:
Disable default or existing connection profiles on the host?

Reply:
Right; this removes the existing interface configuration for the `ens6f0` or `ens5f0` interface.